src/applications/gi/components/vet-tec/VetTecContactInformation.jsx

```diff
@@ -42,16 +42,26 @@ export const VetTecContactInformation = ({ institution }) => (
       </div>
     </div>
     {/* Production flag for 19534 */}
-    {!environment.isProduction() &&
-      institution.schoolCertifyingOfficials[0] && (
-        <div className="additional-information row vads-u-margin-y--4">
-          {VetTecScoContact(
-            institution.schoolCertifyingOfficials[0],
-            'School certifying officials',
-          )}
-          {VetTecScoContact(institution.schoolCertifyingOfficials[1])}
-        </div>
-      )}
+    {!environment.isProduction() && institution.schoolCertifyingOfficials[0] && (
+      <React.Fragment>
+        {institution.schoolCertifyingOfficials.map(
+          (sco, i) =>
+            i % 2 === 0 && (
+              <div className="additional-information row vads-u-margin-y--4">
+                {/* {i === 0 ? () : ()} */}
+                {VetTecScoContact(
+                  sco,
+                  i === 0 && 'School certifying officials',
```

Comment:
since only the first SCO is using the header it might be best to pull the header out of VetTecScoContact and have it be independent of each SCO

Reply:
Yeah that would be a little bit cleaner to do so. I'll have to adjust the styling, but cant imagine those styling changes would be too intensive. It seemed to make enough sense to do it this way when we only were displaying 2 SCOs, but now that there are X number, having that header outside makes a little bit more sense.